Failure when attmpting to use lxc_guest

Running 

yaml used:
```
resource "proxmox_lxc_guest" "gitea_lxc" {
  target_node  = "pve"
  name         = "gitea-lxc"
  
  template {
        file    = "debian-12-standard_12.12-1_amd64.tar.zst"
        storage = "local"
  } 

  password     = var.gitea_lxc_password
  privileged = true
  
  cpu {
    cores = 2
  }

  memory=4096
  swap=2048

  root_mount {
        size    = "100G"
        storage = "local-lvm"
    }


  network {
    id            = 0
    name          = "eth0"
    bridge        = "vmbr0"
    ipv4_dhcp     = true
  }

  mount {
    slot    = "0"
    host_path = "/mnt/lxc_shares/deb_nas"
    guest_path      = "/mnt/nas"
  }

  ssh_public_keys = var.public_key
}
```
Error Received:
```
5:28:26 PM
Plan: 1 to add, 0 to change, 0 to destroy.
5:28:26 PM
proxmox_lxc_guest.gitea_lxc: Creating...
5:28:26 PM
╷
5:28:26 PM
│ Error: Request cancelled
5:28:26 PM
│ 
5:28:26 PM
│   with proxmox_lxc_guest.gitea_lxc,
5:28:26 PM
│   on main.tf line 3, in resource "proxmox_lxc_guest" "gitea_lxc":
5:28:26 PM
│    3: resource "proxmox_lxc_guest" "gitea_lxc" {
5:28:26 PM
│ 
5:28:26 PM
│ The plugin.(*GRPCProvider).ApplyResourceChange request was cancelled.
5:28:26 PM
╵
5:28:26 PM
5:28:26 PM
Stack trace from the terraform-provider-proxmox_v3.0.2-rc07 plugin:
5:28:26 PM
5:28:26 PM
panic: runtime error: slice bounds out of range [2:1]
5:28:26 PM
5:28:26 PM
goroutine 71 [running]:
5:28:26 PM
github.com/Telmate/terraform-provider-proxmox/v2/proxmox/Internal/resource/guest/lxc/mounts.sdkMount(0x1, {0xc001273da0, 0x1, 0x0?})
5:28:26 PM
	github.com/Telmate/terraform-provider-proxmox/v2/proxmox/Internal/resource/guest/lxc/mounts/sdk_mount.go:18 +0x1f4f
5:28:26 PM
github.com/Telmate/terraform-provider-proxmox/v2/proxmox/Internal/resource/guest/lxc/mounts.SDK(0x1, 0xc000232a20)
5:28:26 PM
	github.com/Telmate/terraform-provider-proxmox/v2/proxmox/Internal/resource/guest/lxc/mounts/sdk.go:12 +0xcc
5:28:26 PM
github.com/Telmate/terraform-provider-proxmox/v2/proxmox.lxcSDK(0x1, _)
5:28:26 PM
	github.com/Telmate/terraform-provider-proxmox/v2/proxmox/resource_lxc_guest.go:334 +0x40c
5:28:26 PM
github.com/Telmate/terraform-provider-proxmox/v2/proxmox.resourceLxcGuestCreate({0x1094218, 0xc00022a930}, 0xc000232a20, {0xd75220?, 0xc000942c30})
5:28:26 PM
	github.com/Telmate/terraform-provider-proxmox/v2/proxmox/resource_lxc_guest.go:100 +0x1a8
5:28:26 PM
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc00057b400, {0x1094170, 0xc000b68db0}, 0xc000232a20, {0xd75220, 0xc000942c30})
5:28:26 PM
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.37.0/helper/schema/resource.go:849 +0x119
5:28:26 PM
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc00057b400, {0x1094170, 0xc000b68db0}, 0xc000891520, 0xc000232360, {0xd75220, 0xc000942c30})
5:28:26 PM
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.37.0/helper/schema/resource.go:980 +0xb47
5:28:26 PM
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc00000e0c0, {0x1094170?, 0xc000b68cf0?}, 0xc000b0ab40)
5:28:26 PM
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.37.0/helper/schema/grpc_provider.go:1484 +0x1085
5:28:26 PM
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc00022e140, {0x1094170?, 0xc000b68210?}, 0xc00094b600)
5:28:26 PM
	github.com/hashicorp/terraform-plugin-go@v0.28.0/tfprotov5/tf5server/server.go:941 +0x3b9
5:28:26 PM
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0xf05720, 0xc00022e140}, {0x1094170, 0xc000b68210}, 0xc00094b580, 0x0)
5:28:26 PM
	github.com/hashicorp/terraform-plugin-go@v0.28.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:687 +0x1a6
5:28:26 PM
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00020a400, {0x1094170, 0xc000b68180}, 0xc0009806c0, 0xc000784390, 0x1737fc8, 0x0)
5:28:26 PM
	google.golang.org/grpc@v1.72.1/server.go:1405 +0x1036
5:28:26 PM
google.golang.org/grpc.(*Server).handleStream(0xc00020a400, {0x1094b50, 0xc00073b6c0}, 0xc0009806c0)
5:28:26 PM
	google.golang.org/grpc@v1.72.1/server.go:1815 +0xb88
5:28:26 PM
google.golang.org/grpc.(*Server).serveStreams.func2.1()
5:28:26 PM
	google.golang.org/grpc@v1.72.1/server.go:1035 +0x7f
5:28:26 PM
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 26
5:28:26 PM
	google.golang.org/grpc@v1.72.1/server.go:1046 +0x11d
5:28:26 PM
5:28:26 PM
Error: The terraform-provider-proxmox_v3.0.2-rc07 plugin crashed!
```

--- a/proxmox/Internal/resource/guest/lxc/mounts/schema_mount.go
+++ b/proxmox/Internal/resource/guest/lxc/mounts/schema_mount.go
@@ -27,20 +27,23 @@ func SchemaMount() *schema.Schema {
 								Summary:  schemaID + " cannot be empty",
 								Severity: diag.Error}}
 						}
-						if len(v) > 2 {
-							if v[0:2] != prefixSchemaID {
+						if len(v) > len(prefixSchemaID) {
+							if v[0:len(prefixSchemaID)] != prefixSchemaID {
 								return diag.Diagnostics{{
 									Summary:  schemaID + " must start with '" + prefixSchemaID + "'",
 									Severity: diag.Error}}
 							}
-							num, err := strconv.ParseUint(v[2:], 10, 64) // validate that the rest is a number
+							num, err := strconv.ParseUint(v[len(prefixSchemaID):], 10, 64) // validate that the rest is a number
 							if err != nil || num > maximumID {
 								return diag.Diagnostics{{
 									Summary:  "invalid " + schemaID,
 									Severity: diag.Error}}
 							}
+							return nil
 						}
-						return nil
+						return diag.Diagnostics{{
+							Summary:  schemaID + " must start with '" + prefixSchemaID + "'",
+							Severity: diag.Error}}
 					}},
 				schemaType: {
 					Type:     schema.TypeString,

